extensions/objects/templates/math-expression-content-editor.component.ts

```diff
@@ -57,7 +57,7 @@ export class MathExpressionContentEditorComponent implements OnInit {
     // part of an editable list).
     this.svgString = '';
     this.numberOfElementsInQueue = 0;
-    this.value.mathExpressionSvgIsBeingProcessed = true;
+    this.value.mathExpressionSvgIsBeingProcessed = !this.value.svg_filename;
```

Comment:
Can you please explain this change. I'm not quite sure I understand it.
Also a minor thing -- shouldn't `svg_filename` be camel-cased?

Reply:
Added a comment to explain this.

I think `svg_filename` is fine here because it is how this property (and `raw_latex`) is defined in the customisation args specs based on `rich_text_components_definitions.ts` and `objects.py`. This seems to be the pattern followed by other objects in general i.e. property name matches that in the object customisation args definition and the corresponding frontend component.

Reply:
@kevintab95 Can you explain more clearly why the previous code didn't work (or is working fine in Chrome but not Firefox)?

Reply:
Hi @seanlip this particular code change is related to math expression not autopopulating when trying to edit an existing component in the editor (it fixes (1) mentioned here: https://github.com/oppia/oppia/issues/13094#issuecomment-861792360)

The fix for (2) is lines 101-105.

Reply:
I have updated this code to be a bit more self explanatory.